collections/_article/gdc-2023-retrospective.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "GDC 2023: Retrospective"
+excerpt: "We attended the Game Developers Conference (GDC) in San Francisco this year."
+categories: ["events"]
+author: Clay John
+image: /storage/blog/covers/gdc-2023-retrospective.jpg
+date: 2023-04-05 14:00:00
+---
+
+For the first time ever Godot had a booth at the Game Developer Conference in San Francisco. In the past we never considered having a booth as the cost is totally prohibitive for us. However, this year the booth was graciously donated by W4 Games, without them, it would not have been possible to have the kind of presence we had.
+
+To give you a taste of what the booth (and our experience at GDC) was like, here is a short video prepared by Emilio Coppola which summarizes our experience this year.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/vdAwlI6NG0I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+Edited by Emi. Music track: [Builder - QUEEN JAZZ](https://queenjazz.bandcamp.com/album/bops) (Used with permission).
+
+In this post we would like to discuss our experience from the perspective of the project and highlight what went well and what we could have improved.
```

Comment:
```suggestion
In this post, we would like to discuss our experience from the perspective of the project and highlight what went well and what we could have improved.
```

Reply:
As explained above, there shouldn't be a comma here